sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFlux.java

```diff
@@ -58,6 +58,18 @@ public PagedFlux(Supplier<Mono<PagedResponse<T>>> firstPageRetriever,
         this.nextPageRetriever = nextPageRetriever;
     }
 
+    /**
+     * Creates an instance of {@link PagedFlux} using argument as a supplier that fetches the first page of {@code T}.
+
+     * <p><strong>Code sample</strong></p>
+     * @param firstPageRetriever Supplier that retrieves the first page
+     */
+    public PagedFlux(Supplier<Mono<PagedResponse<T>>> firstPageRetriever) {
```

Comment:
Can we call the other constructor from here? PagedFlux(firstPageRetriever, nextUrl -> Mono.empty());